website/components/News.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { useRouter } from 'next/router';
+import facepaint from 'facepaint';
+import ArrowRightIcon from '@leafygreen-ui/icon/dist/ArrowRight';
+import { css } from 'emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { Overline, Subtitle, H2 } from '@leafygreen-ui/typography';
+import { spacing, breakpoints } from '@leafygreen-ui/tokens';
+
+const mq = facepaint(
+  Object.values(breakpoints).map(bp => `@media (min-width: ${bp}px)`),
+  { literal: true },
+);
+
+const newsContainer = css`
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding-bottom: ${spacing[4]}px;
+
+  ${mq({
+    marginTop: [`${spacing[4]}px`, 'unset'],
+    marginBottom: [`${spacing[4]}px`, 'unset'],
+    paddingLeft: [`${spacing[4]}px`, 'unset'],
+  })}
+`;
+
+const subtitleStyle = css`
+  cursor: pointer;
+  text-decoration: none;
+  font-weight: bolder;
+  display: inline-flex;
+  align-items: center;
+
+  &:hover {
+    & > svg {
+      opacity: 1;
+      transform: translate3d(3px, 0, 0px);
+    }
+  }
+`;
+
+const iconStyle = css`
+  transform: translate3d(-3px, 0, 0px);
+  transition: all 100ms ease-in;
+
+  ${mq({
+    opacity: [1, 1, 0, 0],
+    marginLeft: [`${spacing[2]}px`, `${spacing[2]}px`, 0],
+  })}
+`;
+
+const overlineMargin = css`
+  margin-top: 28px;
+`;
+
+const overlineColor = css`
+  color: ${uiColors.gray.dark1};
+`;
+interface UpdateProps {
+  date: string;
+  story: string;
+  path?: string;
+  href?: string;
+}
+
+function Update({ date, story, path, href }: UpdateProps) {
+  const { push } = useRouter();
+  let renderedStory: React.ReactElement | null;
+
+  if (path) {
+    renderedStory = (
+      <Subtitle as="p" className={subtitleStyle} onClick={() => push(path)}>
+        {story}
+        <ArrowRightIcon className={iconStyle} />
+      </Subtitle>
+    );
+  } else {
+    renderedStory = (
+      <Subtitle as="a" className={subtitleStyle} href={href}>
+        {story}
+        <ArrowRightIcon className={iconStyle} />
+      </Subtitle>
+    );
+  }
+
+  return (
+    <div className={overlineMargin}>
+      <Overline className={overlineColor}>{date}</Overline>
+      {renderedStory}
+    </div>
+  );
+}
+
+function News() {
+  return (
+    <div className={newsContainer}>
+      <H2 as="h1">Whats New</H2>
+      <Update
```

Comment:
it would be more scalable for the stories to not be stored all in one file. if adding a new story was as simple as writing a markdown file, then even non-eng could easily add new ones.

Reply:
I don't have access to file reading methods from inside of components that don't exist  in the pages dir. I can move this there and then read data from a static file if you think that makes more sense. Unless you had something else in mind?

Reply:
This component doesn't need to move. It can receive the data it needs from the page through props.

Reply:
Updated